### PR TITLE
simplify the matrix build

### PR DESCRIPTION
**Pull request by chrisns**
only adds things to the matrix that need to be done

### CODE REVIEW

File: .github/workflows/dockerbuild.yml [hunk 1]
```diff
@@ -13,37 +13,46 @@ jobs:
...
+        wget -q http://old.openzwave.com/downloads/ -O downloads.html
+
+        EXISTING_TAGS=$(wget -q https://registry.hub.docker.com/v1/repositories/chrisns/openzwave/tags -O -)
+
+        MATRIX="{\"openzwave\":[\"git\""
+
+        for i in {2..4}
+        do
+          VERSION=$(cat downloads.html | ./pup '#pagecontent > table:nth-child(2) > tbody > tr:nth-child('${i}') > td:nth-child(1) text{}')
+          if [[ $EXISTING_TAGS != *"$VERSION"* ]]
```
Comment (robertsLando) [inline .github/workflows/dockerbuild.yml:31]: Does this actually work??

Comment (chrisns) [reply .github/workflows/dockerbuild.yml:31]: I mean I tested it, but I'm not going to warranty it till I've seen it run in anger 😃

Comment (robertsLando) [reply .github/workflows/dockerbuild.yml:31]: Seems right: https://stackoverflow.com/questions/229551/how-to-check-if-a-string-contains-a-substring-in-bash

Nice to know :)

**Review by robertsLando — APPROVED** [review]
LGTM

**Pull request merged**